jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1120,40 +1120,41 @@ def _precision_config_proto(precision: Optional[Tuple[PrecisionType, PrecisionTy
   proto.operand_precision.append(int(precision[1]))
   return proto
 
-# _try_tf_conv returns a Tensor when it succeeds, or a string describing why
-# it did not succeed otherwise.
 def _try_tf_conv(lhs, rhs, window_strides, padding, lhs_dilation, rhs_dilation,
                  dimension_numbers, feature_group_count, batch_group_count,
-                 out_shape) -> Union[str, TfVal]:
+                 out_shape) -> TfVal:
+
+  class ConvError(Exception):
+    def __init__(self, e):
+      super().__init__("Couldn't convert conv_general_dilated without XLA: "
```

Comment:
I would align this with _xla_path_disabled_error, so that all primitives that cannot be converted with enable_xla=False use the same message pattern.

In #6722 I was going to change the error message globally, and add the ability to add additional messages. That PR got rolled back, because of some issues to XlaPad. I expect to be able to re-land it in a couple of day.

Reply:
Alright, perhaps it then makes sense to wait for your PR to land before adding this one?

Reply:
#6755 got merged. Inspired by your change I have added to it the surfacing of the precise reason why convolution cannot be converted. PTAL.

(I hope this won't get rolled back again)